ops/terraform/env/mgmt/base_config/values/mgmt.yaml

```diff
@@ -56,6 +50,6 @@ bfd:
Txa0F2BFkHEkTNRTHN4eCtCAZbpQKSD0JzAMTrGgofdfJunM3S0MDpwfM0qt77KxAMVXsbe_8OwkAVjtBwzDxGWsBo2oc-_Ub72y4li4Hj_S5--z9H2VnFXYNfldT5SmrB5coA3hx8y5elujW1V-fkPjZfKrEliNmSmPQ081irFeD9s_Q0TftAG9NDMN8wkgspinRdMZCqwTJcPQAV7QJLVEpyOMKpi4y53xrEgaZzcLmInGXOosMReJ2F_wO4oHdW5yB8Skjnw5ThAXRrrOD5Ius0142p1hh5_219xaImRUT1pkdqrPmQUwsqhMxTz6u5SzEtQhilYhMX9jctmXhJjCQeY1vyNSzmXQ_gG7S0e57KD12MqBWqCJlN5YY7Tuj5mpRDyuI_QnRjCWtrphRAVAC3HKeSV3HBce-mICWxhQeMPBooJPTJZo5HZdvGmjmYF1uyG6LjbPkCs9foRT3R8B3UM-pK95AJanmRZxPsVD1rs2DjGupg5OTq9jky5hSkD0wHuxKg3H6rzK9bN7ZHKbW1YqpypjVNqTwGmB4MnpxuYnYR_J2eL3yJXyxLsSAp_PFmCZipL1CSNQD-8q2JiAgf8opkEC-5YR0A3dnkPFfoZx_cY1SRg4bSZ_Sa4gHtm8XOg9zgedfzHpk2g01MdL8bB75tLgT5D4SgyotP4eSySJpyQADLQ2J1kEOzW08YVJBcTo1blcm0JPlQGUIF64eDNzVY4wCoIYGoXXS0iVJzZwcE86ojzG1zP0DZnSNhsgT7dMhQYVgMRYEJY-VzhxRyGPPlxxaIRHQF3zaEgsXcbYbG97BRnkQYPsKBZyTGI7UW5yakQae3-DeqcZu7ETVr5z8Uwj1GbfIqjQrjIwkTKDh7qy0ClOn-pMES6tC_dFlgaVul9WJJBYHz5F-Gyhp6USAAmTjJp-ueTA2ZX2zqWNcjNzvGV550IA0Zm6rL2YoFNMBAWF4Vhm-njbaA8F7k4FKa7y4nR3lIWAr_TU-1RehDCRzjyLx_ACC8mY0U9coApWn3L4NoxZ68nzZ6Tln8iPR8T0X6L0DYbNAJrbIIYw_T7e8EljSfLsBpMCZZXktes5rDwuQv3Lx4HdJmPzTgHfIP0im8xyUm8CC-J6S0FHDwMIB-6AYJieO12MmdMVztmfvBoMFzjgyOGJvf6Pk3vJAEmLfePHX1znZkhKjJ9xQLCntHylhyR9JEQy3cs6PnCdxfsJ_BKe_0Bcup6TZny8fjQehfqipd2m_Ccyjv55SmJqffQWLM2NS7uVT4btA4E0TKCar1JJzlqJQOnlfUR5Bx4n31fY0Bu6VuS4le7-ueiImYkP_i3GtplWtlAEqSkyZZ64X8lkfykERFwA-TyRbI42j2aW-AW4BLWvAAxljs_IHZTHg3R-MlFV1Xt1w15wq1gQ-kDx4ZqCCILfB3SHGsl4oP6VHa3_UNNKZB4dujLne33aB2l7cxiE0PuGy3rn6_lPGkuGyLFTKdFWckp6oSyGIGL5gu6MlKL6-0wqacf81Jh4V0SOeIwocBq6JIWVInqjK6sWGcvgH1krWoQFzy1mK024Wt8xTOGAHZ4_D9cHsldkuFYUPRcUJkqAUGUQlV6askYyQnNR08AAeGcCsQIaE-mIhg9AXj7BJsXWmChHFPJN-K3YQLWdUQIS9yWNJC_iCzjM6Mqg0u3Xd25w4bU1eUMIo0FNiX-0mH3JZhk-grjk3vcmtKN6l2UiLJ34OEO2z5eAEZHu1TdlZyW9XnXwOMy0Mprcnp6WY0cdru_j1OqQckw3lJI1kVLdkEQ8cX2oX6vpVg54Fi2DqSV0ECaaBrz1bk5Wkf5aHf-J97Hnu7s3OYAsZ9ElSsfA-RHe8CP9JCKavbkLvsqNAe3IEK6fC-TR26Y568Jnh1IZJy68TmVdjs9YG-jzg1ks3K6xq_-I-5mn_q5QmteGTkt-vHKD3W0TB2UZjtTVEn6fUamehN5K1cSL-bmcJPLNgLpgkV6YZyUQeBNZ-UWkh3hALVlacGelc2gV51zKd0Wm3MSy5u5eVCDRnfHVw5DEEf8uVSW53Ii0P-NSdcmJIE0W2HBhV1n-bz0rKJ7YTxQHCZH3yjwRF1wMZl-bzrflv0qzIRlLlplfJ3dFzlAteWy_KPvgYfivTY8GnPBC_ffmxf-MfI_PdsNH_Dm7Z31AvUm2ZG8vw-OARhXeplAXtf61O0NUKgmPy48Tl7m_gZUFVtsvHLvJIXMRpCOOiugP1Ica89rNh36rWK1llOewqyaS_pOU_Ta1DU8RPaq49MajEvI1Jm2hneAPpiaH6EoGqXzjjuFRN8EdpIi3cX9ImwDzoh46oBJD_8JqjCwrvyo0ub2luOCjArFuk4d1_WCKo0uajg8kdxOfZl72vv7UOrtivq2uRqxSL0JTk3dHlrOnkEKmr8O2Txw2wqGuQHc5lqIl_3JLZM_5_ohEI316lBLWjSpuY7HF8h0Njqxkrf6f9tCRqpj4H0_mcjeDVfj1rz07OihjTRrSieF3mcvUf5hasMwWjhPgAxPSFCxZZUfE5eT297wRTeBqYj-C1B2hZk2ZjePGDNiUdJMH_JBa5KBWFcHAOg8FOHgj-xqywFJma-sJUy2j9iXNXrrEMYz-THm3p7aJQ1nuEQjAS-VG3X0M0VaGAPdrJgR_mh5bGZ1GZ19g0o1_5smeWhWmvSNiXdBljY0NfWbpu9fnTwJFvvWie0QCYXM2IsC8djId4rYWI2gHFeJsIBv7bfh2U0nJfU4xNGv5RwP_E0qzVFlsmiPSSwJHkBtOg7L7yi2dtvUH7d9p5UEd2KUM7gD5u23cqhYXnDKwWmsx0lfZCx_hBgi7Q7OIAZzBlAjEAhVOUyTdxVvpcqqhILhcDcRDhUS2yJYeNwL1sSzAKRXLiGUbEaMkGEMUCkKS8-p8CAjA_pvu3ADiMd10ARfW8qyHohU8Klzfo_iirmzz13E48mVhcoHOryn3Sq197SguLJY8=<</CIPHER>>
```

Comment:
@dondevun can you also remove adrian.jones@adhocteam.us and kirsten.hansen@adhocteam.us? They were on the ad hoc team on BB2

Reply:
Done!